src/main/java/org/generationcp/middleware/pojos/Germplasm.java

```diff
@@ -240,6 +244,7 @@ public class Germplasm extends AbstractEntity implements Serializable, Cloneable
 	private Integer mgid;
 
 	@OneToMany(mappedBy = "germplasm", fetch = FetchType.LAZY, cascade = CascadeType.ALL)
+	@BatchSize(size = 5000)
```

Comment:
@lik-83  is this batch size intentional?

Reply:
It was not necessary, so I removed it.